espnet/asr/pytorch_backend/asr.py

```diff
@@ -167,8 +168,10 @@ def update_core(self):
 
         # Get the next batch ( a list of json files)
         batch = train_iter.next()
-        # self.iteration += 1 # Increase may result in early report, which is done in other place automatically.
-        x = tuple(arr.to(self.device) for arr in batch)
+        self.iteration += 1
+
+        x = tuple(arr.to(self.device) if arr is not None else None
```

Comment:
Can you leave the comment for this change?
What is the intention?

Reply:
ditto